registration/forms.py

```diff
@@ -39,6 +40,13 @@ class Meta:
         model = User
         fields = (UsernameField(), "email")
 
+    def clean_username(self):
+        username = self.cleaned_data.get('username')
```

Comment:
`username = self.cleaned_data.get('username', '')` since we call `lower()` in the line below